spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerProperties.java

```diff
@@ -28,10 +28,15 @@
  * @author Mark Fisher
  * @author Ilayaperumal Gopinathan
  */
-@ConfigurationProperties(prefix = "deployer.local")
+@ConfigurationProperties(prefix = LocalDeployerProperties.LOCAL_PROPERTIES)
 public class LocalDeployerProperties {
 
 	/**
+	 * Top level prefix for local deployer configuration properties.
+	 */
+	public static final String LOCAL_PROPERTIES = "spring.cloud.deployer.local";
```

Comment:
Given this is always referred using `LocalDeployerProperties`, we can probably name this as `PREFIX`.